app/services/layout/layout-data.ts

```diff
@@ -37,49 +37,49 @@ type ILayoutData = {
 
 export const LAYOUT_DATA: ILayoutData = {
   [ELayout.Default]: {
-    resizeDefaults: { bar1: 156, bar2: 240 },
+    resizeDefaults: { bar1: 20, bar2: 40 },
```

Comment:
why the numbers are so different. Are they percentages now?

Reply:
whoops those changes werent meant to be carried over, will fix